polaris-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -148,6 +177,8 @@ export function Tooltip({
         padding={padding}
         borderRadius={borderRadius}
         zIndexOverride={zIndexOverride}
+        instant={!shouldAnimate}
+        keyboardShortcut={keyboardShortcut}
```

Comment:
Can the suffix markup currently rendered in `TooltipOverlay` be rendered next to the `content` here instead of having a separate prop?